taskiq/abc/broker.py

```diff
@@ -234,15 +234,22 @@ def inner(
                 if inner_task_name is None:
                     fmodule = func.__module__
                     if fmodule == "__main__":  # pragma: no cover
-                        fmodule = ".".join(
-                            sys.argv[0]
-                            .removesuffix(
-                                ".py",
+                        if sys.version_info < (3, 9):
+                            fmodule = ".".join(
+                                remove_suffix(sys.argv[0], ".py").split(
+                                    os.path.sep,
+                                ),
+                            )
+                        else:
+                            fmodule = ".".join(
+                                sys.argv[0]
+                                .removesuffix(
+                                    ".py",
+                                )
+                                .split(
+                                    os.path.sep,
+                                ),
```

Comment:
Hello! What do you think about not splitting the logic into two parts and using only a method that is suitable for all versions of python?

Reply:
Yes, it's great 😌